grunttasks/xssmitigate.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var gumbo = require('gumbo-parser');
+var sets = require('simplesets');
+
+module.exports = function (grunt) {
+  grunt.registerTask('xss-flag', 'Flag potentially harmful files', [
+    // grunt addition for htmllint
+    'clean',
+    'selectconfig:dist',
+    'l10n-generate-pages',
+    'l10n-json-to-html',
```

Comment:
Since this task is 80% shared w/ `l10n-htmllint` not sure if it makes sense to abstract the shared setup into a new task. Or not, whatever. I can't think of a good name for a task that does:
- 'clean'
- 'selectconfig:dist'
- 'l10n-generate-pages'
- 'l10n-json-to-html'